apps/server/src/modules/tool/launch-tool/error/parameter-not-implemented.loggable-exception.ts

```diff
@@ -0,0 +1,19 @@
+import { NotImplementedException } from '@nestjs/common';
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+
+export class ParameterNotImplementedLoggableException extends NotImplementedException implements Loggable {
```

Comment:
rename to ParameterTypeNotImplementedLoggableException to be a litle bit more clear what exactly is not implemented